simulationdataschema/model_system_review.py

```diff
@@ -0,0 +1,864 @@
+            return None
+
+        # Lattice vectors
+        if self.lattice_vectors is not None:
+            ase_atoms.set_cell(self.lattice_vectors.to("angstrom").magnitude)
+            if self.lattice_vectors_reciprocal is None:
+                self.lattice_vectors_reciprocal = (
+                    2 * np.pi * ase_atoms.get_reciprocal_cell() / ureg.angstrom
+                )
+        else:
+            logger.info("Could not find `AtomicCell.lattice_vectors`.")
+
+        return ase_atoms
+
+    def normalize(self, archive, logger):
+        # If the labels and atomic_numbers are not specified, return with an error
+        if self.labels is None and self.atomic_numbers is None:
+            logger.error(
+                "Could not read `AtomicCell.labels` or `AtomicCell.atomic_numbers`."
+            )
+            return
+        atomic_labels = self.labels
+        atomic_numbers = self.atomic_numbers
+
+        # Labels
+        if atomic_labels is None and atomic_numbers is not None:
+            try:
+                atomic_labels = [
+                    ase.data.chemical_symbols[number] for number in atomic_numbers
+                ]
+            except IndexError:
+                logger.error(
+                    "The `AtomicCell.atomic_numbers` are out of range of the periodic table."
+                )
+                return
+        self.labels = atomic_labels
+
+        # Use ASE Atoms functionalities to extract information about the AtomicCell
+        ase_atoms = self.to_ase_atoms(logger)
+
+        # Atomic numbers
+        if atomic_labels is not None and atomic_numbers is None:
+            atomic_numbers = ase_atoms.get_atomic_numbers()
+        self.atomic_numbers = atomic_numbers
+
+        # We then normalize `GeometricSpace`
+        super().normalize(archive, logger)
+
+
+class Symmetry(ArchiveSection):
+    """
+    A base section used to specify the symmetry of the `AtomicCell`. This information can
+    be extracted via normalization using the MatID package, if `AtomicCell` is specified.
+    """
+
+    bravais_lattice = Quantity(
+        type=str,
+        description="""
+        Bravais lattice in Pearson notation.
+
+        The first lowercase letter identifies the
+        crystal family: a (triclinic), b (monoclinic), o (orthorhombic), t (tetragonal),
+        h (hexagonal), c (cubic).
+
+        The second uppercase letter identifies the centring: P (primitive), S (face centered),
+        I (body centred), R (rhombohedral centring), F (all faces centred).
+        """,
+    )
+
+    hall_symbol = Quantity(
+        type=str,
+        description="""
+        Hall symbol for this system describing the minimum number of symmetry operations,
+        in the form of Seitz matrices, needed to uniquely define a space group. See
+        https://cci.lbl.gov/sginfo/hall_symbols.html. Examples:
+            - `F -4 2 3`,
+            - `-P 4 2`,
+            - `-F 4 2 3`.
+        """,
+    )
```

Comment:
> in the form of Seitz matrices

is a bit misleading, since it sounds like we're storing those matrices... I'd just remove it.